CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## Main
 
+* [5187](https://github.com/grafana/loki/pull/5187) **aknuds1** Rename metric `cortex_experimental_features_in_use_total` to `loki_experimental_features_in_use_total` and metric `log_messages_total` to `loki_log_messages_total`.
```

Comment:
I think this would break some alerts. Consider adding `loki_experimental_features_in_use_total` in addition to the other alert.

Reply:
@jeschkies where should it (they?) be added? Not sure what you're requesting :)

Reply:
@jeschkies I think, like @sandeepsukhani [already mentioned](https://github.com/grafana/loki/pull/5187#pullrequestreview-858050384).  This rename doesn't need any update in any alerts (double checked in our clusters, plus it's not used in our OSS mixin as well).

So adding it in changelog is sufficient.

Reply:
@aknuds1 I found `log_messages_total` in one of the doc, which we may have to rename as well.
https://github.com/grafana/loki/blob/main/docs/sources/operations/observability.md?plain=1#L17

Reply:
@kavirajk thanks, updated observability.md.

Reply:
Great. I will ask some maintainers to merge it as soon the build passes. Thanks for the patience :pray:

Reply:
Cool! Just happy to finally get this work merged :)